client/lib/oauth-store/test/index.js

```diff
@@ -88,16 +87,20 @@ describe( 'oAuthStore', function() {
 		} );
 	} );
 
-	it( 'sets OAuth token when login is correct', function() {
-		var before = global.document;
-
-		global.document = { location: { replace: sinon.stub() } };
+	it( 'sets OAuth token when login is correct', sinon.test( function() {
+		const originalDocument = global.document,
+			replaceSpy = this.spy();
 
-		sinon.stub( oAuthToken, 'setToken' );
+		global.document = {
+			location: {
+				replace: replaceSpy
```

Comment:
Done :) It's a bit slower, but test file looks much clean in my opinion. Let me know if that works for you.